### DISCUSSION TITLE
Building a massive 1000 GB TTS torrent

### DISCUSSION THREAD

**Discussion by neurlang** [coqui-ai/TTS#2597] [Ideas]
Hello I have a new disk
I want to build a massive FLAC dataset containing various languages (as many as possible)
This will not be a repack of ljspeech or whatever, focusing more on more rare data sets that could disappear.
Anyone have an idea of what to include (please also mention the estimated size of the data in GB).
Also idea of a file name / meta data convention would be great.

**Comment by FrontierDK** [comment]
Each dataset will most likely have it's own unique rights, so instead of creating a massive file, create several list files with each dataset - with usage rights listed next to each file?

**Comment by neurlang** [comment]
I wasn't thinking about a massive file, mostly the opposite, one torrent with language subfolders and then speaker subfolders.
This will ensure that people can download just the thing they need.
Speaking of the rights to the dataset, obviously only things that are reasonable public shareable licensed will be distributed.
